tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,26 @@
+"""
+Handles logging functionality.
```

Comment:
Logging is done at many places and calling another function will increase the overhead I guess. Second at each logging instance the payload structure is not same. Third this could have been done inside the event class but it was not as it was not recommended in the spec discussion.